svm/src/lib.rs

```diff
@@ -473,6 +475,13 @@ impl LiteSVM {
                 let mut account_found = true;
                 let account = if solana_sdk::sysvar::instructions::check_id(key) {
                     construct_instructions_account(message)
+                } else if ed25519_program::check_id(key) || secp256k1_program::check_id(key) {
```

Comment:
It seems like these accounts should just be added at startup. We could add a `with_precompile` to the builder